batch/journey.test.js

```diff
@@ -241,6 +243,171 @@ describe("batch importing", () => {
   it("tears down and cleans up", () => cleanup(client));
 });
 
+describe("batch deleting", () => {
+  const client = weaviate.client({
+    scheme: "http",
+    host: "localhost:8080",
+  });
+
+  it("sets up schema", () => setup(client));
+  it("sets up data", () => setupData(client));
+
+  it("batch deletes with dryRun and verbose output", () =>
+    client.batch
+      .objectsBatchDeleter()
+      .withClassName(thingClassName)
+      .withWhere({
+        operator: Operator.EQUAL,
+        valueString: "bar1",
+        path: ["stringProp"]
+      })
+      .withDryRun(true)
+      .withOutput(Output.VERBOSE)
+      .do()
+      .then(result => {
+        expect(result.dryRun).toBe(true);
+        expect(result.output).toBe(Output.VERBOSE);
+        expect(result.match).toEqual({
+          class: thingClassName,
+          where: {
+            operands: null, // FIXME should not be received
```

Comment:
is this `FIXME` (and the ones below) still relevant?

Reply:
Apparently weaviate responds `operands: null` even if operands property is not sent in the request.
Not sure it is desired behaviour (other, not set properties are missing). If it is not, then fix should be applied on weaviate side.